### PR TITLE
[IEco] Updated sections 1, 2 and 3

### PR DESCRIPTION
**Pull request by dcaoc03**
Added and developed the following sections of IEco:
- Mercados. Eficiência. Papel do Governo
- PIB e Crescimento Económico, Inovação e Progresso Tecnológico
- Inflação. Desemprego e Desigualdade

I'm planning to update the following four sections as well.
Be sure to look for any typos or other mistakes, I'm open to any feedback.

### CODE REVIEW

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 1]
```diff
@@ -19,11 +19,17 @@ Os conteúdos abaixo encontram-se incompletos e, possivelmente, com alguns erros
 Procuram-se contribuidores.
 :::
 
-## Capitalismo
+## Economia e Capitalismo
 
-A maioria das economias atuais são capitalistas (também chamadas de Economias de Mercado).
+A **Economia** é o estudo das interaçações e das trocas estabelecidas entre várias pessoas e entre
+pessoas e o seu ambiente na produção dos seus meios de subsistência
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:25]: Missing period (`.`) at end of sentence
```suggestion
pessoas e o seu ambiente na produção dos seus meios de subsistência.
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 2]
```diff
@@ -19,11 +19,17 @@ Os conteúdos abaixo encontram-se incompletos e, possivelmente, com alguns erros
...
 
-No capitalismo, a principal forma de organização económica é a [**empresa**](color:green).
+O capitalismo é um exemplo de um **Sistema Económico**, ou seja, uma maneira organizada de gerir
+a economia de uma sociedade (outros exemplos de Sistemas Económicos incluem o Feudalismo, o Esclavagismo
+e o Planeamento Económico Centralizado, este último popularizado pela União Soviética no século XX)
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:29]: Missing period (`.`) at end of sentence
```suggestion
e o Planeamento Económico Centralizado, este último popularizado pela União Soviética no século XX).
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 3]
```diff
@@ -62,53 +68,226 @@ em si, mas o que perdemos por não fazer a alternativa.
...
 
-<!-- TODO Curva da Oferta e da Procura -->
+A procura e a oferta por um bem ou serviço muitas vezes podem ser traduzidas por funções:
+
+- A [**curva da procura**](color:red) corresponde à quantidade que os consumidores estão dispostos a comprar
+  a um determinado preço.
+  - Quanto maior for o preço, mais reduzida será a quantidade procurada.
+- A [**curva da oferta**](color:blue) corresponde à quantidade que as empresas estão dispostas a produzir
+  por um determinado preço
+  - Quanto maior for o preço, mais as empresas quererão produzir
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:86]:
```suggestion
- A [**curva da oferta**](color:blue) corresponde à quantidade que as empresas estão dispostas a produzir
  por um determinado preço.
  - Quanto maior for o preço, mais as empresas quererão produzir.
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 4]
```diff
@@ -62,53 +68,226 @@ em si, mas o que perdemos por não fazer a alternativa.
...
+![Excedentes](./assets/0001-excedentes.png#dark=2)
+
+:::tip[Excedente Total]
+
+$$
+\text{Excedente Total (ET)} = {\text{EC}} + {\text{EP}}
+$$
+
+Corresponde aos ganhos totais do comércio para a sociedade.
+É dado pela da área a sombreado na figura
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:128]:
```suggestion
É dado pela da área a sombreado na figura.
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 5]
```diff
@@ -19,11 +19,17 @@ Os conteúdos abaixo encontram-se incompletos e, possivelmente, com alguns erros
 Procuram-se contribuidores.
 :::
 
-## Capitalismo
+## Economia e Capitalismo
 
-A maioria das economias atuais são capitalistas (também chamadas de Economias de Mercado).
+A **Economia** é o estudo das interaçações e das trocas estabelecidas entre várias pessoas e entre
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:24]:
```suggestion
A **Economia** é o estudo das interações e das trocas estabelecidas entre várias pessoas e entre
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 6]
```diff
@@ -62,53 +68,226 @@ em si, mas o que perdemos por não fazer a alternativa.
...
+
+No entanto, quando um dos fatores mencionados acima não é respeitado, originam-se as chamadas **Falhas de Mercado**.
+
+São exemplos de Falhas de Mercado:
+
+- [**Externalidades**](color:blue)
+- [**Informação Assimétrica**](color:red)
+- [**Contratos Incompletos**](color:yellow)
+- [**Abuso de Bens Públicos**](color:green)
+- [**Exitência do Poder de Mercado**](color:orange)
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:210]:
```suggestion
- [**Existência do Poder de Mercado**](color:orange)
```

File: content/ieco/0001-mercados-eficiencia-governo.md [hunk 7]
```diff
@@ -62,53 +68,226 @@ em si, mas o que perdemos por não fazer a alternativa.
...
+2. Encontrar e calcular custos exatos;
+3. Decisões judiciais;
+4. Fundos limitados e incapacidade de pagar compensações.
+
+Outra solução consiste na imposição de **Políticas Governamentais**, tais como:
+
+1. **Regulação da Produção**: colocar um limite ao nível da quantidade socialmente ótima;
+2. **Imposto/Subsídio Pigouviano**: aplicado sobre as empresas que geram efeitos negativos/positivos;
+   - o valor do imposto corresponde ao valor do **CME** no ótimo social;
+3. **Compensação das Partes Afetadas**: pagamento direto de uma indemização aos terceiros envolvidos;
```
Comment (diogotcorreia) [inline content/ieco/0001-mercados-eficiencia-governo.md:252]:
```suggestion
3. **Compensação das Partes Afetadas**: pagamento direto de uma indemnização aos terceiros envolvidos;
```

File: content/ieco/0002-pib-crescimento-economico.md [hunk 8]
```diff
@@ -36,7 +70,76 @@ $$
...
+## Revolução Industrial
+
+A **Revolução Industrial** foi uma mudança de paradigma de produção que consistiu na transição de uma mão-de-obra industrial maioritariamente manual para uma mecanizada. Teve origem no Reino Unido no início do século XVIII.
+
+Alguns fatores que podem explicar o começo deste movimento neste local são:
+
+- mão-de-obra cara face a energia barata;
+- atributos culturais (como o trabalho árduo e a poupança);
+- a revolução científica associada ao Iluminismo ;
+- a abudância de carvão nas colónias.
```
Comment (diogotcorreia) [inline content/ieco/0002-pib-crescimento-economico.md:140]:
```suggestion
- a abundância de carvão nas colónias.
```

File: content/ieco/0002-pib-crescimento-economico.md [hunk 9]
```diff
@@ -55,8 +158,41 @@ Pode ser de um de dois tipos:
...
+
+![Renda da Inovação](./assets/0002-renda-da-inovacao.png#dark=2)
+
+### Mercado de Trabalho
+
+O progresso tecnológico também pode causar:
+
+- melhoria dos padrões de vida a longo prazo;
+- aumento do desemprego a curto prazo;
+  - pode também haver um aumento do emprego (através de realocação de mão-se-obra, por exemplo).
```
Comment (diogotcorreia) [inline content/ieco/0002-pib-crescimento-economico.md:173]:
```suggestion
  - pode também haver um aumento do emprego (através de realocação de mão-de-obra, por exemplo).
```

File: content/ieco/0002-pib-crescimento-economico.md [hunk 10]
```diff
@@ -55,8 +158,41 @@ Pode ser de um de dois tipos:
...
+
+A criação de emprego é altamente **pró-ciclica** (mais procura, mais vagas, mais trabalho), enquanto que a destruição de emprego é **contra-cíclica**.
```
Comment (diogotcorreia) [inline content/ieco/0002-pib-crescimento-economico.md:175]:
```suggestion
A criação de emprego é altamente **pró-cíclica** (mais procura, mais vagas, mais trabalho), enquanto que a destruição de emprego é **contra-cíclica**.
```

File: content/ieco/0003-inflacao-desemprego-desigualdade.md [hunk 11]
```diff
@@ -20,46 +20,133 @@ Procuram-se contribuidores.
...
+**Subida generalizada dos nível de preços** numa economia.
+:::
 
-Mas como podemos medir a inflação?
+### Medir a Inflação
 
-O **Índice de Preços no Comsumidor** (IPC) mede o nível geral de preços que os
-consumidores pagam por bens e serviços (incluindo impostos),
-baseado num "cabaz representativo de bens de consumo".
+**[Índice de Preços no Comsumidor (IPC)](color:red)**: mede o nível geral de preços que os
```
Comment (diogotcorreia) [inline content/ieco/0003-inflacao-desemprego-desigualdade.md:29]:
```suggestion
**[Índice de Preços no Consumidor (IPC)](color:red)**: mede o nível geral de preços que os
```

File: content/ieco/0003-inflacao-desemprego-desigualdade.md [hunk 12]
```diff
@@ -20,46 +20,133 @@ Procuram-se contribuidores.
...
+- Aumento do poder de negociação dos trabalhadores sobre as empresas (por exemplo, na presença de pleno emprego).
+- Aumento dos custos de produção.
+
+### Inflação e Emprego
+
+Maior oferta de emprego pode estar associada a maior inflação:
+
+1. Maior oferta de emprego (Menos desemprego).
+2. Aumento do poder de negociação dos trabalhadores.
+3. Aumeto dos salários.
```
Comment (diogotcorreia) [inline content/ieco/0003-inflacao-desemprego-desigualdade.md:117]:
```suggestion
3. Aumento dos salários.
```

File: content/ieco/0002-pib-crescimento-economico.md [hunk 13]
```diff
@@ -36,7 +70,76 @@ $$
...
+Choques ocorrem tanto em sociedades industrializadas como agrárias.
+
+São exemplos de choques os fenómenos climáticos extremos e desastres naturais (secas, cheias, furacões, sismos, ...), as guerras, as pandemias, entre outros.
+
+:::
+
+Perante um choque, os consumidores têm tendência para **suavizar o consumo**, podendo recorrer às seguintes estratégias:
+
+- **[Auto-seguro (Self-insurance)](color:yellow)**: poupar dinheiro ou pedir um empréstimo, sacrificando poder de compra futuro;
+- **[Co-seguro (Co-insurance)](color:red)**: recorrer a apoios da Segurança Social ou do Governo.
```
Comment (diogotcorreia) [inline content/ieco/0002-pib-crescimento-economico.md:90]:
```suggestion
- **[Auto-seguro (_Self-insurance_)](color:yellow)**: poupar dinheiro ou pedir um empréstimo, sacrificando poder de compra futuro;
- **[Co-seguro (_Co-insurance_)](color:red)**: recorrer a apoios da Segurança Social ou do Governo.
```

File: content/ieco/0003-inflacao-desemprego-desigualdade.md [hunk 14]
```diff
@@ -20,46 +20,133 @@ Procuram-se contribuidores.
...
+5. Aumento dos preços.
+
+**[Curva de Phillips](color:purple)**: estabelece uma relação entre a taxa de inflação e a taxa de emprego.
+
+- Quando o desemprego está abaixo do equilíbrio, há uma **pressão positiva** sobre os preços e os salários, a **taxa de inflação aumenta**.
+- Quando o desemprego está acima do equilíbrio, há uma **pressão negativa** sobre os preços e os salários, a **taxa de inflação diminui**.
+
+![Curva de Phillips](./assets/0003-curva-de-phillips.png#dark=2)
 
 <!-- TODO consequências inflação e deflação -->
```
Comment (diogotcorreia) [inline content/ieco/0003-inflacao-desemprego-desigualdade.md:128]: Isn't this done now?

File: content/ieco/0003-inflacao-desemprego-desigualdade.md [hunk 15]
```diff
@@ -20,46 +20,133 @@ Procuram-se contribuidores.
...
 
-## Estatísticas do Mercado de Trabalho
+## Mercado de Trabalho
+
+O mercado de trabalho de trabalho divide a população nas seguintes categorias:
+
+- População em idade ativa (população com idade entre 16 e 70 anos)
+  - População Ativa (Labour Force)´
```
Comment (diogotcorreia) [inline content/ieco/0003-inflacao-desemprego-desigualdade.md:135]:
```suggestion
  - População Ativa (Labour Force)
```

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Found some typos, but overall awesome work, thanks a lot!

Btw, if you want to remove the "Página Incompleta" banner at the top of the pages feel free to do so. I put it there since I wrote these pages in a rush in the day of the exam as self-study.

**Comment by dcaoc03** [comment]
Thanks for the help! I'll be sure to correct the typos and remove the "Página Incompleta" banner.

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**